pynndescent/rp_trees.py

```diff
@@ -443,7 +443,6 @@ def sparse_euclidean_random_projection_split(inds, indptr, data, indices, rng_st
     return indices_left, indices_right, hyperplane, hyperplane_offset
 
 
-@numba.jit()
```

Comment:
This annotation was removed since numba can't do the type inference for recursive calls (in this case at least). The underlying computation (in `euclidean_random_projection_split`) is still JIT'd though.